src/Core/System.php

```diff
@@ -236,10 +236,27 @@ public static function currentLoad()
 		return max($load_arr[0], $load_arr[1]);
 	}
 
+	/**
+	 * Redirects to another URL and exits this process.
+	 *
+	 * @param string $toUrl The destination URL (Default is empty, which is the default page of the Friendica node)
+	 * @param bool $ssl if true, base URL will try to get called with https:// (works just for relative paths)
+	 */
+	public static function redirect($toUrl = '', $ssl = false)
```

Comment:
Shouldn't it be in App since it invokes `App->getBaseURL`? What's the point of `System` as a separate static class if it uses non-static `App` methods?

Reply:
You're right, there are now two ways to go:
- Remove the `$a->getBaseURL()` and require it as parameter (so it's real static)
- Move it to `$a->redirect();`
What's your preference?

Reply:
It's so nice when people reach the same conclusions as yourself on their own. `$a->redirect()` it is.